VideoLocker/src/test/java/org/edx/mobile/test/PresenterTest.java

```diff
@@ -0,0 +1,32 @@
+package org.edx.mobile.test;
+
+import android.support.annotation.NonNull;
+
+import org.edx.mobile.view.Presenter;
+import org.junit.After;
+
+import static org.mockito.Mockito.mock;
+
+public abstract class PresenterTest<P extends Presenter<V>, V> extends BaseTest {
+    protected P presenter;
+    protected V view;
+
+    protected final void startPresenter(@NonNull P p) {
```

Comment:
Should this just have a `@Before` annotation added, as it seems to be invoked from all the test methods?